src/scss/blocks/_browser-compat.scss

```diff
@@ -69,6 +69,16 @@
     color: get-utility-value('color', 'state-warn-text');
   }
 
+  &[data-compat='flag'] {
```

Comment:
We could theoretically collapse these into the existing `preview` and `no` styles, but I figured we might want to style them separately at some point?